src/Cake.Core/Cake.Core.csproj

```diff
@@ -131,11 +131,21 @@
     <Compile Include="IO\IRegistry.cs" />
     <Compile Include="IO\IRegistryKey.cs" />
     <Compile Include="IO\Machine.cs" />
+    <Compile Include="IO\NuGet\AssemblyFilePathFrameworkNameParser.cs" />
```

Comment:
Depends on whether you care about using namespaces to denote "components" (where you'd typically allow types to use other types that are higher in the namespace hierarchy, and prohibit using those that are lower) versus simply grouping code functionally.  I'm open to whatever you prefer.